pom.xml

```diff
@@ -19,7 +19,7 @@
   </parent>
 
   <artifactId>apk-parent</artifactId>
-  <version>0.0.1</version>
+  <version>0.0.0</version>
```

Comment:
FWIW, I'd like to encourage we start using a "-SNAPSHOT" suffix on the version numbers. (I've updated the nexus-format-archetype to do this).
The ORB we use for releasing will handle versions like 0.0.1-SNAPSHOT. One downside is you have to remember to manually bump the version to the next development SNAPSHOT after doing a release.

Reply:
In: c2b77ee

Reply:
Based on conversation with @DarthHater, we are going to reverse this change and open an issue to revisit